pkg/store/proxy.go

```diff
@@ -81,38 +81,73 @@ func (s *ProxyStore) Info(ctx context.Context, r *storepb.InfoRequest) (*storepb
 		StoreType: s.component.ToProto(),
 	}
 
-	MinTime := int64(math.MaxInt64)
-	MaxTime := int64(0)
-
+	minTime := int64(math.MaxInt64)
+	maxTime := int64(0)
 	stores := s.stores()
+
+	if len(stores) == 0 {
+		res.MaxTime = math.MaxInt64
```

Comment:
I am not sure about this. I know there is not regression to previous version (same logic), but does this make sense? ;p
Let's leave the comment maybe? `// Edge case: we have all of the data if there are no stores.` and add some `TODO`. 

Essentially not sure if we no store we should do either
* max time = MaxInt64 and min time = maxInt64 as nothing is there.
or
* max time = MaxInt64 and min Time MinInt64 if we want to have `whole range` ;p 

But we should not spend time on this in this PR I guess.

Reply:
Would an issue be more appropriate than a TODO in code maybe?

Reply:
I'm adding the "Edge case:" comment back and will open an issue to decide this as a follow up.

Reply:
Thanks, makes sense